src/v2/Apps/Auction/Components/RegistrationForm.tsx

```diff
@@ -90,9 +90,9 @@ const InnerForm: React.FC<InnerFormProps> = props => {
         <Box mx="auto">
           <ConditionsOfSaleCheckbox
             selected={values.agreeToTerms}
-            onSelect={value => {
-              setFieldValue("agreeToTerms", value)
+            onSelect={(value) => {
               setFieldTouched("agreeToTerms")
+              setFieldValue("agreeToTerms", value)
```

Comment:
And this is the other fix.

Reply:
Did you see this issue on the pure register flow? I haven't seen it before

Reply:
nvm! I'm wrong -- it's there